java/adapter/avro/src/test/java/org/apache/arrow/AvroToArrowTest.java

```diff
@@ -425,4 +427,13 @@ public void testNullableUnionType() throws Exception {
     checkPrimitiveResult(expected, vector);
   }
 
+  @Test
+  public void testValueVectorDataPopulator() {
```

Comment:
It would be better if this was an actually useful test for Avro, but at least could you add a comment that this is meant to only test out the dependence on the vector tests artifact? Otherwise, it's a little confusion why this test is here at all.

Reply:
Thanks, I added a comment to clarify this.

Reply:
Sorry, this is kind of bugging me having a pointless test here. How about moving the dependency to arrow.tools and change this test to use it? https://github.com/apache/arrow/blob/master/java/tools/src/test/java/org/apache/arrow/tools/EchoServerTest.java#L157

Reply:
ok, fixed.